CompetitionTemplate/src/competition/operator_interface/OperatorCommandMap.java

```diff
@@ -187,4 +190,14 @@ public void setupDashboard(RotateTowardsBallCommand rotate, CollectForwardBallCo
         collect.includeOnSmartDashboard();
         acquire.includeOnSmartDashboard();
     }
+    
+    @Inject
+    public void switchEnableSafeArmOperation(
+            OperatorInterface oi,
+            ArmSubsystem arm,
+            EnableSafeArmOperationCommand enableSafeArm,
+            DisableSafeArmOperationCommand disableSafeArm){
+        oi.operatorPanelButtons.getifAvailable(1).whenPressed(enableSafeArm);
```

Comment:
Since we want 1 switch to control these, you should instead put them both on the same button but one can be whenReleased and the other whenPressed